### PR TITLE
chore (CI): ensure support to py312

### PR DESCRIPTION
**Pull request by johnnv1**
related to kornia/workflows/pull/21

### CODE REVIEW

File: .github/workflows/pr_test_cpu.yml [hunk 1]
```diff
@@ -20,32 +20,32 @@ jobs:
...
-    uses: kornia/workflows/.github/workflows/mypy.yml@v1.9.1
+    uses: kornia/workflows/.github/workflows/mypy.yml@v1.10.0
 
   tutorials:
-    uses: kornia/workflows/.github/workflows/tutorials.yml@v1.9.1
+    uses: kornia/workflows/.github/workflows/tutorials.yml@v1.10.0
 
   docs:
-    uses: kornia/workflows/.github/workflows/docs.yml@v1.9.1
+    uses: kornia/workflows/.github/workflows/docs.yml@v1.10.0
```
Comment (johnnv1) [inline .github/workflows/pr_test_cpu.yml:48]: review this one, maybe the docs deps do not support 3.12 yet

**Review by edgarriba — APPROVED** [review]

**Comment by johnnv1** [comment]
numpy 2.0 release breaks some things here, but is unrelated within this patch

**Comment by edgarriba** [comment]
> numpy 2.0 release breaks some things here, but is unrelated within this patch

should be fix with numpy <2.0 ?

**Comment by johnnv1** [comment]
> > numpy 2.0 release breaks some things here, but is unrelated within this patch
> 
> should be fix with numpy <2.0 ?

probably, or update the tests

**Pull request merged**